### ISSUE TITLE
Embedded timelines widget ID info box

### ISSUE THREAD

**Issue by JanrikV** [rainlab/twitter-plugin#22]
So you can't find any widget ID in your twitter account settings page.

"Widgets
Embedded timelines can no longer be managed through account settings on twitter.com. Going forward, you must configure your embeds on publish.twitter.com."

**Comment by LukeTowers** [comment]
@JanrikV so does the documentation need to be changed?

**Comment by JanrikV** [comment]
> @JanrikV so does the documentation need to be changed?

Yes, and you don't find any widget ID in profile settings page anymore, so Embed Timeline component doesn't work.

**Comment by LukeTowers** [comment]
@JanrikV did you go to publish.twitter.com to find a widget ID there?

**Comment by JanrikV** [comment]
> @JanrikV did you go to publish.twitter.com to find a widget ID there?

Yes, but I don't found any widget ID in there.

**Comment by LukeTowers** [comment]
@JanrikV if you want you can resubmit https://github.com/rainlab/twitter-plugin/pull/16 although you'd need to make sure it was backwards compatible (i.e. you can't just remove the existing properties like the original PR did.
